package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "brainstem-js",
+  "repository": "https://github.com/mavenlink/brainstem-js",
+  "filename": "brainstem",
+  "standalone": "Brainstem",
+  "version": "0.4.5-1",
+  "author": {
+    "name": "Mavenlink",
+    "email": "dev@mavenlink.com"
+  },
+  "license": "MIT",
+  "main": "lib/brainstem.js",
+  "scripts": {
+    "prepublish": "gulp build-module",
+    "test": "gulp test"
+  },
+  "dependencies": {
+    "backbone": "^1.1.2",
+    "inflection": "^1.8.0",
+    "jquery": "^2.1.4",
+    "underscore": "^1.4.4"
+  },
+  "devDependencies": {
+    "babel": "^6.5.2",
+    "babel-core": "^6.7.2",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "backbone-factory": "https://github.com/jrolfs/Backbone-Factory.git#master",
```

Comment:
Heh, excuse me, your are correct. I did not publish it yet. Ideally I think we would switch to a maintained Backbone factory implementation but for the time being I should probably open a PR on this to try and get the status quo published on NPM.